host-configs/lassen-blueos_3_ppc64le_ib_p9-clang@10.0.1.2_cuda.cmake

```diff
@@ -109,7 +109,7 @@ set(MFEM_DIR "${TPL_ROOT}/mfem-4.4.0" CACHE PATH "")
 
 set(HDF5_DIR "${TPL_ROOT}/hdf5-1.8.22" CACHE PATH "")
 
-set(LUA_DIR "/usr/WS1/axom/libs/blueos_3_ppc64le_ib_p9/2022_11_09_10_19_47/clang-10.0.1.1/lua-5.4.4" CACHE PATH "")
+set(LUA_DIR "/usr/WS1/axom/libs/blueos_3_ppc64le_ib_p9/2022_12_01_14_30_45/clang-10.0.1.1/lua-5.4.4" CACHE PATH "")
```

Comment:
@white238 -- is there a difference between `clang-10.0.1.1` and `clang-10.0.1.2` ?
I suppose this is another concretization error from spack?

We should probably switch over to the new `require` clause in our package specs -- https://spack.readthedocs.io/en/latest/build_settings.html#package-requirements

Reply:
We just need to update uberenv and this will go away. They changed the default from "fresh" to "reuse" a while back and I fixed it in uberenv but apparently never updated our uberenv.  Basically Spack decided that the other clang's lua was good enough and used it instead of rebuilding from scratch.

Reply:
There is no difference between the two compilers. One is the CUDA build and one is not. It is so we can have two different directories and not conflict.